src/config/rds_mysql.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "RDS",
+  "displayName": "Amazon RDS",
+  "rules": [
+    {
+      "aws": {
+        "logGroup": "/aws/rds/instance/%{DATA:resource_id}/%{GREEDYDATA:log_type}"
+      },
+      "sources": [
+        {
+          "sourceType": "logs",
+          "source": "log_group",
+          "condition": "$prefix('/aws/rds/instance/')"
+        }
+      ],
+      "attributes": [
+        {
+          "key": "aws.service",
+          "pattern": "'rds'"
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "resource_id"
+        },
+        {
+          "key": "aws.arn",
+          "pattern": "format_arn('arn:{}:rds:{}:{}:db:{}', [partition, region, account_id, resource_id])",
+          "priority": 10
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "dt_meid_rds_v2(\"aws.arn\")"
+        },
+        {
+          "key": "severity",
+          "pattern": "if(log_type == 'slowquery' || contains(log_content, '[Warning]'), &'WARNING', &if(contains(log_content, '[Error]'), &'ERROR', &'INFO', @), @)"
```

Comment:
From `examples/rds.txt` it looks like slowquery logs have `logGroup` ending in `/general` but in `tests/unit/logs/test_transformation_full.py` the `logGroup` name ends with `/slowquery`. Which one is correct?

Reply:
Also, I remember one discussion in our team that we'd like to avoid using `contains` due to performance reasons and only use `starts_with`. @bmrozinski-dt I think it was related to one of your PRs. Do you remember the details?

Reply:
We decided to take a safe path and we determine severity only based on the `starts_with` condition and avoid the `contains` because of performance (single log content can contain even thousands of characters). We haven't done any performance tests though. So maybe `contains` is not such a risk...
If there is a way to check only the first 50 characters (for example) - maybe it could be a more efficient way.

Reply:
@mswiatkowska already also had this discussion with @mmajcher I believe, and did profiling of contains for those particular log type as a result. I expected she came back with the results, but it would make sense to have them publicly too and available to all - hopefully she can confirm when she comes back. 

What she told me was, for those logs characteristics:
* with fixed schema (as in the examples) that also limits expected average length
* (my assumption) a more consistent test sample (possibly unlike the profiling I already did in CloudTrail before, that was deemed insufficient and failed to persuade you)
* 10k logs > contains is more expensive by 0,6s (3.2s vs 2.6s)
* 1k logs > contains is more expensive by 0.1s

For me this difference is worth the results here.

Reply:
Yes I discussed this with @mswiatkowska and we decided that it seems safe.
I still wish I had timing estimates for 10MB and 50MB (log volume per minute).

10k with 3.2s seems fine but we can do 50k max (15s) AND I believe this perf test was done on super fast computer not on little 1vthread lambda so there is still some uncertainty. So 15s might grow considerably but same applies to scenario without this "contains". Guess we'll see :) and we might prioritize perf improvments once again if we see shortcomings in production scenarios

Reply:
With performance you're never sure. In case this makes your mind a little easier, I think my PC is newer/faster than Maria's and when I did the tests both locally and in AWS, AWS was just mildly slower.